app/javascript/components/App.js

```diff
@@ -6,16 +6,16 @@ import {
   Route,
   Switch
 } from "react-router-dom"
-import { Nav, NavItem } from "reactstrap" 
+import { Nav, NavItem } from "reactstrap"
 import Header from "./components/Header"
 class App extends React.Component {
   render() {
 
     return (
-        <Router>
-          <Header {...this.props}/>
+      <Router>
+        <Header {...this.props} />
         <Switch>
-           {/* <Route exact path="/" component={ Home } /> 
+          {/* <Route exact path="/" component={ Home } /> 
```

Comment:
We shouldn't check in commented out code.

Reply:
Deleted code we weren't using.